terraform/modules/azure-jenkinsinfra-storage/variables.tf

```
@@ -0,0 +1,28 @@
+# Required variables
+variable "service_fqdn" {
+  type = string
+}
+
+variable "storage_service_principal_ids" {
```

See my comment in https://github.com/jenkins-infra/azure/pull/557/files#r1431730896 which underline the poor naming I did on the controller module.

WDYT about the following name in this case to make it easier to understand?

```suggestion
variable "storage_active_directory_application_owners" {
```